Vizzuality/half-earth-v3
# Feature/nrc layer controls

## Add NRC page layer controls
### Description
- Add layer controls to the protected areas and additional protection layers on NRC country page
- Show missing protected areas layer on map
- Fix country selection crash

### Testing instructions
Go to a country NRC page. Scroll the sidebar and you should see the controls for the two shown layers.

### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-246?atlOrigin=eyJpIjoiNzFiZDcyYzRmZGJlNDI1YzgwMDQxNGE0NDhiYTAxZjQiLCJwIjoiaiJ9
